DOCS/faq.md

```diff
@@ -36,127 +12,108 @@ standardized short identifier, the full name, the license text, and a
 
-<br>
+The SPDX-legal team leads are listed here: https://spdx.dev/about/governance/
 
-### <span id="exception"></span>Where are license exceptions? Why are they listed separately?
+##  What does it mean when a license ID is "deprecated"?
+A license identifier is "deprecated" when there is an updated license identifier and the deprecated license identifier. SPDX aims to not changes license identifiers, unless there is a compelling reason and is not a decision taken lightly. Deprecated license identifiers, while remaining valid, should no longer be used. The URL to each deprecated license is retained and those license pages are updated to note the deprecation. 
 
-For ease of finding, [license exceptions are listed on their own
-page](https://spdx.org/licenses/exceptions-index.html). SPDX 2.0
-introduced the concept of license expressions and moved the license
-exceptions to a sub-list, in order to accommodate a more realistic
-combination of licenses and exceptions. Exceptions are to be used with
-the WITH operator. For more information about license expressions, see
-[Annex D of the SPDX
-Specification](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/).
+# How do I request adding a license to the SPDX License List?
 
-##### <a href="#top"> Back to Top</a>
+Follow the instructions here:
+https://github.com/spdx/license-list-XML/blob/main/DOCS/request-new-license.md
 
-<br>
 
-### <span id="or"></span>How does one represent a file or package that is disjunctively licensed (i.e., a license choice)?
+## How does one handle licenses not found in the SPDX License List?
 
-Disjunctive licensing can be represented via a license expression using
-the OR operator. For example, a file that is disjunctively licensed
+When a license identified in the software package is not found in the
+list of approved SPDX licenses, one can define a new license label, using `LicenseRef-<name>`..This is explained in [Clause 10 of the SPDX Specification](https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/).
+
+# What are license expressions?
+Sometimes a single license can be used to represent the licensing terms of a package or file, but there are situations where a single license identifier is not sufficient. [SPDX License Expressions](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/) provide a way to construct more complex licensing scenarios. expressions. A license expression could be a single license identifier found on the SPDX License List; a user defined license reference denoted by the LicenseRef-<idString>; a license identifier combined with an SPDX exception; or some combination of license identifiers, license references and exceptions constructed using a small set of defined operators (e.g., AND, OR, WITH and +)   
```

Comment:
Needs backticks around `LicenseRef-<idString>`, otherwise it doesn't render correctly in MarkDown due to the angle brackets

Reply:
fixed